### DISCUSSION TITLE
Support file paths for mesh imports

### DISCUSSION THREAD

**Discussion by lkeselman** [foxglove/community#826] [Q&A]
Feature request: `file` paths for meshes are also able to be loaded by foxglove studio.

**Comment by jtbandes** [comment]
I believe this may work already if you're using the desktop app - cc @achim-k

**Reply by lkeselman** [reply]
Thanks. It does, if the file exists on the machine which is running the desktop app. This request is to send `file` path meshes over foxglove-bridge so only the machine where the bridge is running needs to have the file locally available.

**Reply by achim-k** [reply]
Yes, Studio currently only requests meshes with `package://` over foxglove websocket. From https://github.com/foxglove/ros-foxglove-bridge/issues/264#issuecomment-1715582506:

> The bridge supports it (if you adapt the regex as you mentioned) but Foxglove Studio currently only requests `package://` URL meshes from foxglove bridge. Could you change the mesh `file://` URLs to `package://` URLs? It also seems that your file paths are wrong, or are your meshes located under `/meshes` (in the root directory) ?

@jtbandes should we also attempt to load `file://` URLs over the websocket connection?
